docs/installation/manual/installing_tomcat_and_cantaloupe.md

```diff
@@ -0,0 +1,152 @@
+
+The resultant location of the java JRE binary (and therefore, the correct value of `JAVA_HOME` when it’s referenced) will vary based on the specifics of the machine it’s being installed on; that being said, you can find its exact location using `update-alternatives`:
+
+```bash
+update-alternatives --list java
+```
+
+### Creating a `tomcat` User
+
+Apache Tomcat, and all its processes, will be owned and managed by a specific user for the purposes of keeping parts of the stack segregated and accountable.
+
+```bash
+sudo addgroup tomcat
+sudo adduser tomcat --ingroup tomcat --home /opt/tomcat --shell /usr/bin
+```
+
+You will be prompted to create a password for the `tomcat` user; all the other information as part of the `adduser` command can be ignored.
+
+### Downloading and Placing Tomcat 8
+
+Tomcat 8 itself can be installed in several different ways; while it’s possible to install via `apt-get`, this doesn’t give us a great deal of control over exactly how we’re going to run and manage it; as a critical part of the stack, it is beneficial for our purposes to have a good frame of reference for the inner workings of Tomcat.
+
+We’re going to download the latest version of Tomcat to `/opt` and set it up so that it runs automatically. Bear in mind that with the following commands, this is going to be entirely relative to the current version of Tomcat 8, which we’ll try to mitigate as we go.
+
+```bash
+cd /opt
+sudo wget -O tomcat.tar.gz TOMCAT_TARBALL_LINK
+sudo tar -zxvf tomcat.tar.gz
+sudo mv /opt/TOMCAT_DIRECTORY/* /opt/tomcat
+sudo chown -R tomcat:tomcat /opt/TOMCAT_DIRECTORY
+```
+- `TOMCAT_TARBALL_LINK`: No default can be provided here; you should navigate to the [Tomcat 8 downloads page](https://tomcat.apache.org/download-80.cgi) and grab the link to the latest `.tar.gz` file under the “Core” section of “Binary Distributions”. It is highly recommended to grab the latest version of Tomcat 8, as it will come with associated security patches and fixes.
+- `TOMCAT_DIRECTORY`: This will also depend entirely on the exact version of tomcat downloaded - for example, `apache-tomcat-8.5.47`. Again, `ls /opt` can be used to find this.
+
+### Creating a setenv.sh Script
+
+When Tomcat runs, some configuration needs to be pre-established as a series of environment variables that will be used by the script that runs it.
+
+`/opt/tomcat/bin/setenv.sh | tomcat:tomcat/755`
+```
+export CATALINA_HOME="/opt/tomcat"
+export JAVA_HOME="PATH_TO_JAVA_HOME"
+export JAVA_OPTS="-Djava.awt.headless=true -server -Xmx1500m -Xms1000m"
+```
+- `PATH_TO_JAVA_HOME`: This will vary a bit depending on the environment, but will likely live in `/usr/lib/jvm` somewhere (e.g., `/usr/lib/jvm/java-8-openjdk-amd64` for an installation on a machine with an AMD processor); again, in an Ubunutu environment you can check a part of this using `update-alternatives --list java`, which will give you the path to the JRE binary within the Java home
+
```

Comment:
Just a suggestion, not a requirement, the SOLR install text indicates you should configure your system's open file limit "to 65000 to avoid operational disruption". A short description of how to do that so it persists can be found [on Medium](https://medium.com/@muhammadtriwibowo/set-permanently-ulimit-n-open-files-in-ubuntu-4d61064429a).